build-tests/api-extractor-scenarios/etc/spanSorting/api-extractor-scenarios.api.json

```diff
@@ -378,6 +385,7 @@
               "text": "(o: {\n    a: number;\n    b(): string;\n}) => void"
             }
           ],
+          "sourceFile": "src/spanSorting/index.ts",
```

Comment:
I don't normally worry too much about optimization, but the duplication of `"sourceFile": "src/spanSorting/index.ts"` is somewhat concerning because it is a potentially long string, and there are so many redundant copies.  In https://github.com/microsoft/rushstack/issues/895#issuecomment-1223348529 I suggested that we could eliminate much of this duplication by only serializing this string when it is different from the parent API item.  Maybe it is worth implementing that in this PR.

Reply:
Done!